src/screens/Ticketing/Ticket/TicketInfo.tsx

```diff
@@ -169,12 +204,10 @@ const TicketInfoTexts = (props: TicketInfoViewProps) => {
           {tariffZoneSummary}
         </ThemeText>
       )}
-      {showTravelCardActiveWarning && (
-        <View style={styles.notInspectableWarning}>
-          <ThemeIcon svg={Warning} style={styles.notInspectableWarningIcon} />
-          <ThemeText isMarkdown={true}>
-            {t(TicketTexts.ticketInfo.travelcardIsActive)}
-          </ThemeText>
+      {warning && (
```

Comment:
@Gøran Dalum  I haven't put this under any flag/condition like "show only for new tokens" or something. Is there anything like that, which should to be considered?

Reply:
Not that I can think of now